__tests__/Unit/Components/Tasks/SuggestionList.test.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { fireEvent, getByText, render, screen } from '@testing-library/react';
+import SuggestionList from '@/components/tasks/SuggestionBox/SuggestionList';
+import { GithubInfo } from '@/interfaces/suggestionBox.type';
+import { DUMMY_PROFILE as placeholderImageURL } from '@/constants/display-sections';
+
+const gitInfo: GithubInfo[] = [
+    {
+        github_id: 'Jan',
+        profileImageUrl: placeholderImageURL,
+    },
+    {
+        github_id: 'John',
+        profileImageUrl: placeholderImageURL,
+    },
+];
+
```

Comment:
this also please

Reply:
@Pratiyushkumar Sorry, we cannot. as we have no fixture folder.

Reply:
__mocks__/db is your fixture folder, 
https://stackoverflow.com/questions/12071344/what-are-fixtures-in-programming